src/rules/require-hook.ts

```diff
@@ -27,12 +31,28 @@ const isNullOrUndefined = (node: TSESTree.Expression): boolean => {
   );
 };
 
-const shouldBeInHook = (node: TSESTree.Node): boolean => {
+const isExcludedFnCall = (
+  node: TSESTree.CallExpression,
+  options: RequireHooksOptions,
+): boolean => {
+  const nodeName = getNodeName(node);
+
+  if (nodeName === null) {
+    return false;
+  }
+
+  return !!options.excludedFunctions?.includes(nodeName);
```

Comment:
You should be able to just cast this to a `string`, which makes the code a little smaller without any real loss (since this is a known TS limitation that we've got already in a few other places)

```suggestion
  return !!options.excludedFunctions?.includes(getNodeName(node) as string));
```

Reply:
Thanks! This is my first encounter with TS.

Reply:
No problem, you're doing good!